data/serializer/src/main/java/tech/pegasys/teku/api/migrated/BlockRewardData.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.migrated;
+
+public class BlockRewardData {
+  private final String proposerIndex;
+  private final String total;
+  private final String attestations;
+  private final String syncAggregate;
+  private final String proposerSlashings;
+  private final String attesterSlashings;
+
+  public BlockRewardData(
+      String proposerIndex,
+      String total,
+      String attestations,
+      String syncAggregate,
+      String proposerSlashings,
+      String attesterSlashings) {
```

Comment:
We'll ultimately want these as uint64 or long I think... i know the resulting type is string but we don't want to have to do string conversion for object creation...

Reply:
this will alter object creation but shouldnt alter the data in the json definition